src/borg/archiver.py

```diff
@@ -2916,6 +2916,17 @@ def define_borg_mount(parser):
 
         For FUSE configuration and mount options, see the mount.fuse(8) manual page.
 
+        The man page references ``user_id`` and ``uid``, which are not the same
+        thing. ``user_id`` is set by libfuse (or the filesystem) and contains
+        the effective user id of the mount owner. ``uid`` on the other hand
+        refers to the actual owner of the file. While you can pass ``uid`` as a
+        mount option to borg to change the owner of all of the files (i.e,
+        ``borg mount -o uid=1000,gid=1000``) you can use the ``--numeric-ids``
+        switch instead.
+
+        ``numeric-ids`` will use the ``uid``s and ``gid``s from the archive
+        instead of attempting to map username and groupnames to local ids.
+
```

Comment:
user/group: the default behaviour is to use the archived user/group names of each file and map them via the system's name-to-id mapping to the respective user/group ids. Alternatively, you can use ``numeric-ids`` (XXX check whether there is only --numeric-ids cli cmd option or also numeric-ids fstab option XXX) to use the archived user/group ids and not try to map from names.

The mount.fuse man page talks about ``user_id`` and ``group_id`` mount options (implemented by libfuse) which specify the user/group id of the mount owner (XXX what does that even mean? XXX). They are set automatically, you do not need to specify them (XXX could one specify them differently? XXX).

The ``uid`` and ``gid`` mount options (implemented by borg) are something different: you can use them to override user/group id of all files instead of using the archived user/group id.

Reply:
Quite some unclarity (just wrote what I had in mind, didn't look up all the stuff / try it).

The key here is just telling what option causes what behaviour precisely. In order of usefulness / frequency of use.

Reply:
Alright I implemented your suggestions. According to https://github.com/libfuse/libfuse/blob/bdd2d4110fbc6d2059eb699efad2cca4a7eacccb/doc/kernel.txt mount owner just means "user who does the mounting". There is no definition for "filesystem owner", but extrapolating just means something similar. Looked at fstab options, don't really think there is a direct correspondence to `numeric-ids`.